### ISSUE TITLE
[BUG] Black Screen with GPU + KDE

### ISSUE THREAD

**Issue by Ashkaan** [linuxserver/docker-webtop#274]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I just get a black screen when accessing the webtop when I use any distro with KDE.  I can't control anything or make anything happen on screen.

### Expected Behavior

I should see a desktop.

### Steps To Reproduce

docker compose up

### Environment

```markdown
- OS: TrueNAS 24.10
- How docker service was installed: built-in
```

### CPU architecture

x86-64

### Docker creation

```bash
name: webtop
services:
  webtop:
    container_name: webtop
    deploy:
      resources:
        reservations:
          devices:
            - driver: nvidia
              count: all
              capabilities:
                - compute
                - video
                - graphics
                - utility
    environment:
      - PUID=3000
      - PGID=3000
      - TZ=America/Los_Angeles
      - TITLE=Ashkaan's Fedora
    image: lscr.io/linuxserver/webtop:ubuntu-kde
    network_mode: bridge
    ports:
      - 3000:3000
    restart: unless-stopped
    security_opt:
      - seccomp:unconfined
    shm_size: 1gb
    volumes:
      - /mnt/pool/docker/webtop:/config
      - /etc/resolv.conf:/etc/resolv.conf:ro
networks:
  bridge: {}
```
```


### Container logs

```bash
_XSERVTransmkdir: ERROR: euid != 0,directory /tmp/.X11-unix will not be created.
```

**Comment by dfldylan** [comment]
I have the same issue. After cancel nvidia setup, this issue disappear

**Comment by dfldylan** [comment]
#243 same here

**Comment by thelamer** [comment]
Does debian-kde work with the same settings?

**Comment by Ashkaan** [comment]
No, it doesn't.  In fact, neither does ubuntu-xfce anymore, which did work on a previous version.

**Comment by Ashkaan** [comment]
Hey team, this container is still not writing for me.  What can we do to fix the issue?

**Comment by real-limitless** [comment]
I was able to work around this issue by doing a `podman exec -it <container_name> bash` and manually executing the startwm.sh 
script inside `/defaults` 

Another way I was able to work around this issue was by starting the desktop wm manually as well.

**Comment by mg-dev25** [comment]
The black screen is not a bug, it is a normal behavior implement by kasmvnc to indicate the GPU driver is not correct. Often this is because, when using KDE, the host for nvidia card, don't use the open source driver.

**Comment by Xitee1** [comment]
Oh I think I'm not having the exact same issue, this here is related to GPU. I do not even try to use a GPU. It just takes really long to start displaying a picture on this specific host and I can't figure out what is wrong. There's nothing in the logs that differs from the other hosts. Super annoying having to wait 5+ minutes after doing a restart of the container.

**Comment by mg-dev25** [comment]
@Xitee1 You need to share the yml docker compose file, for us to help you.

**Comment by thelamer** [comment]
Images have been completely overhauled, if you still have an issue open a new one.
